MaterialLib/Fluid/Density/createFluidDensityModel.cpp

```diff
@@ -0,0 +1,56 @@
+/*!
+   \file  createFluidDensityModel.cpp
+   \brief create an instance of a fluid density class.
+
+   \copyright
+    Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+               Distributed under a Modified BSD License.
+               See accompanying file LICENSE.txt or
+               http://www.opengeosys.org/project/license
+*/
+
+#include "createFluidDensityModel.h"
+
+#include "BaseLib/Error.h"
+
+#include "MaterialLib/Fluid/ConstantFluidProperty.h"
+#include "IdealGasLaw.h"
+#include "LinearTemperatureDependentDensity.h"
+#include "LiquidDensity.h"
+
+namespace MaterialLib
+{
+namespace Fluid
+{
+FluidProperty* createFluidDensityModel(BaseLib::ConfigTree const* const config)
+{
+    //! \ogs_file_param{material__fluid__density__type}
+    auto const type = config->getConfigParameter<std::string>("type");
+
+    if (type.find("constant") != std::string::npos)
```

Comment:
I think `type == "constant"` is better, because `type.find()` also finds substrings of `type`